extensions/sql-migration/src/wizard/subscriptionSelectionPage.ts

```diff
@@ -0,0 +1,124 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import { MigrationWizardPage } from '../models/migrationWizardPage';
+import { MigrationStateModel, StateChangeEvent } from '../models/stateMachine';
+import { SUBSCRIPTION_SELECTION_PAGE_TITLE, SUBSCRIPTION_SELECTION_AZURE_ACCOUNT_TITLE, SUBSCRIPTION_SELECTION_AZURE_PRODUCT_TITLE, SUBSCRIPTION_SELECTION_AZURE_SUBSCRIPTION_TITLE } from '../models/strings';
```

Comment:
I haven't seen us using this naming convention for constants in this codebase, IIRC.  I think we have been doing something like `Constants.SubscriptionSelectionPageTitle`, etc.  Though I do like the old C-style naming. 😄

Reply:
I can change this if you'd like as well!

Reply:
I'll merge this in for now but I can change it whenever!